packages/core/tests/__snapshots__/builder.test.ts.snap

```diff
@@ -162,7 +162,7 @@ exports[`should use rspack as default bundler > apply rspack correctly 1`] = `
         "oneOf": [
           {
             "generator": {
-              "filename": "static/image/[name].[contenthash:8][ext]",
+              "filename": "static/svg/[name].[contenthash:8].svg",
```

Comment:
Is there any reason for this PR to change the order of the asset rules?

Reply:
put the easy-to-hit and high-frequency rule behind (svg, image), which would be hit as early as possible and have better performance...
small change